src/models/embed/index.ts

```diff
@@ -1,70 +1,96 @@
 import { RequestOptions } from '../../core';
 import * as Resources from '../../resources';
+import { PageInfo } from '../interfaces';
+
+export interface EmbeddingMediaMetadata {
+  inputUrl?: string;
+  inputFilename?: string;
+  videoClipLength?: number;
+  videoEmbeddingScope?: string[];
+  duration?: number;
+}
 
 export interface Embedding {
-  float: number[];
+  segments?: SegmentEmbedding[];
+  errorMessage?: string;
+  metadata?: EmbeddingMediaMetadata;
+}
+
+export interface SegmentEmbedding {
+  float?: number[];
```

Comment:
embedding_floats

Reply:
to make it consistent with Py SDK, I think it's better to have a same name `embeddingFloat`

Reply:
sorry EmbeddingsFloat